java/src/org/certificatetransparency/ctlog/serialization/Deserializer.java

```diff
@@ -109,15 +110,15 @@ public static ParsedLogEntry parseLogEntry(InputStream merkleTreeLeaf, InputStre
     MerkleTreeLeaf treeLeaf = parseMerkleTreeLeaf(merkleTreeLeaf);
     LogEntry logEntry = new LogEntry();
 
-    Ct.LogEntryType entryType = treeLeaf.timestampedEntry.getEntryType();
+    Ct.LogEntryType entryType = treeLeaf.timestampedEntry.entryType;
 
     if (entryType == Ct.LogEntryType.X509_ENTRY) {
       Ct.X509ChainEntry x509EntryChain = parseX509ChainEntry(extraData,
-        treeLeaf.timestampedEntry.getSignedEntry().getX509());
+        treeLeaf.timestampedEntry.signedEntry.getX509());
```

Comment:
Weird, I was fighting Emacs the whole time while touching those lines, as it was doing exactly that, but I figured that I didn't have the right Google Java style rules set up, so I kept putting it back the way it was... 😛